content/en/blog/ospo-definition.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "Open Source Program Office (OSPO) Definition"
+author: todogroup
+date: 2020-05-13
+---
+
+The TODO Group is [over 5 years old](https://todogroup.org/blog/welcome/) and as an organization have brought together the brightest in the field of open source program management. We have produced a variety of open source program management [guides](https://todogroup.org/guides/), [tools](https://github.com/todogroup/repolinter), [surveys](https://github.com/todogroup/survey) and even an [awesome list](https://github.com/todogroup/awesome-oss-mgmt) to advance the practice.
```

Comment:
There's a mixture of references to the TODO Group as a singular entity ('is') and a plural entity ('have') here. If we're using the American English way of referring to groups, 'have' should be 'has'. If we're using the British English way, maybe 'is' should be 'are' but I'm not super-confident about that.

Reply:
fixed thx